$MirOS: ports/print/cups/patches/patch-systemv_cancel_c,v 1.1.7.1 2005/03/18 15:51:00 tg Exp $
--- systemv/cancel.c.orig	Wed Feb  8 18:39:33 2006
+++ systemv/cancel.c	Tue May  9 17:39:40 2006
@@ -275,7 +275,7 @@ main(int  argc,				/* I - Number of comm
       }
       else
       {
-        sprintf(uri, "ipp://localhost/jobs/%d", job_id);
+        snprintf(uri, sizeof(uri), "ipp://localhost/jobs/%d", job_id);
 	ippAddString(request, IPP_TAG_OPERATION, IPP_TAG_URI, "job-uri", NULL,
 	             uri);
       }
